### DISCUSSION TITLE
Photoprism + Cryptomator Volume

### DISCUSSION THREAD

**Discussion by jlnkls** [photoprism/photoprism#2230] [Q&A]
Hello,

I am using [Cryptomator ](https://cryptomator.org/) for encrypting my pictures on an external HDD. Let's say that, when connected to my Windows 10 PC, the drive letter is P:\ . The Cryptomator vault that is contained within P:\ , once unlocked, gets revealed as X:\ . Thus, X:\ contains all the pictures that I want to see on Photoprism.

I have tested Photoprism to show some locally stored files (that is, on C:\ ), which has worked successfully. Now, I am updating the Windows 10 [`docker-compose.yml`](https://dl.photoprism.app/docker/windows/docker-compose.yml) file so as to mount the X:\ volume:

```
    volumes:
      # "C:/user/username/folder:/photoprism/folder"       # example
      - "X:/:/photoprism/originals"
```

However, when running `docker-compose up -d` on cmd, I get the following output:

```
ERROR: for photoprism  Cannot create container for service photoprism: The volume does not contain a recognized file system.
Please make sure that all required file system drivers are loaded and that the volume is not corrupted.
ERROR: Encountered errors while bringing up the project.
```

Is there any way I can mount this Cryptomator-revealed vault on X:\ on Photoprism?

**Comment by srett** [comment]
Just a guess, but afaik, docker on windows uses WSL2, i.e. the variant that runs a virtualized Linux Kernel with extra stuff to interact with the host, for example its file system. My guess would be this doesn't play nice with the FS driver that that encryption software is supplying.

Can you run bash in WSL2 and access drive X: successfully? If yes, the problem has to be more specific and you might be able to solve it, but otherwise, probably no dice.

**Comment by jlnkls** [comment]
I am opening [Tabby](https://tabby.sh/) and running Bash on `WSL / docker-desktop`.  When listing the directories, I get the following:

```
DESKTOP:/tmp/docker-desktop-root/mnt/host# ls
c    d    e    g    p    wsl
```

So, P:/, the encrypted drive, is detected, however, X:/ is not. This might then be a mounting issue with Cryptomator on WSL, I will reach out to their team to see further. I will get back if and when I get X:/ to appear on WSL.
